### PR TITLE
Postmortem Template

### PR DESCRIPTION
**Pull request by VoyTechnology**
Added an initial postmortem template.

### CODE REVIEW

**Comment by VoyTechnology** [comment]
Do not mind the dates. I have written it as if its was written in the past. Everyone is welcome to review.

**Comment by VoyTechnology** [comment]
@GoldenBadger, @rbkoffee can you verify the information is correct?

**Comment by GoldenBadger** [comment]
There are a few typos. I don't want to go and label every single one so do you mind just running a spellchecker on it? :stuck_out_tongue:

**Comment by VoyTechnology** [comment]
@GoldenBadger PTAL.

File: docs/postmortems/2016-05-10-Initial-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+## Overview
+Our initial goal as new admins was to develop Rebrick 2.0 - a complete redesign
+of the Redbrick infrastructure to facilitate the change to an distributed
+infrastructure across many sites as the server room was being demolished. We
+have decided to try containers as it is the new hipster thing. After research
+and consideration we have decided to run a CoreOS cluster on 2 newly
+provisioned machines: Icarus and Daedalus, named the Helios Cluster. Initially
+we had aimed to create as systemd managed containers distributed via fleetctl.
```
Comment (GoldenBadger) [inline docs/postmortems/2016-05-10-Initial-Kubernetes.md]: Aimed to create what as systemd?

File: docs/postmortems/2016-05-10-Initial-Kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+When that failed we continued on to Kubernetes. We had the servers talking to
```
Comment (GoldenBadger) [inline docs/postmortems/2016-05-10-Initial-Kubernetes.md]: Why did it fail?

File: docs/postmortems/2016-05-10-Initial-Kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+eachother successfully and pods running between them. We had Gogs, then Gitlab
```
Comment (GoldenBadger) [inline docs/postmortems/2016-05-10-Initial-Kubernetes.md]: "each other"

File: docs/postmortems/2016-05-10-Initial-Kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+running but with storage problems as the 2 machines did not store data on the
+same server. We had troubles getting NFS storage to share the data between the
+pods. Eventually our terms as admins had ended and the Helios cluster was
```
Comment (GoldenBadger) [inline docs/postmortems/2016-05-10-Initial-Kubernetes.md]: "our term as admins ended"

File: docs/postmortems/2016-05-10-Initial-Kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+Initial Kubernetes Setup
+---
+Redbrick Networking Society
+
+Date:   2016-06-10  
+Author: voy, tbolt
```
Comment (GoldenBadger) [inline docs/postmortems/2016-05-10-Initial-Kubernetes.md]: I didn't write this, so maybe change to the following:

```
Author: voy
Involved: voy, tbolt
```

**Comment by VoyTechnology** [comment]
After some discussion with @GoldenBadger, this is not necessarly a postmortem but retrospective. I will remove the 2015-2016 "postmortem" and just leave the template here.

**Comment by VoyTechnology** [comment]
@GoldenBadger PTAL.

**Comment by GoldenBadger** [comment]
Too late I already did.

**Pull request merged**